appinventor/components/src/com/google/appinventor/components/runtime/EventDispatcher.java

```diff
@@ -198,6 +198,17 @@ public static boolean dispatchEvent(Component component, String eventName, Objec
         dispatched = delegateDispatchEvent(dispatchDelegate, eventClosures, component, args);
       }
     }
+    //If the component does not have a binding event, the generic event is handled.
```

Comment:
This approach might also have some issues in the companion due to #822 and #994, where if one creates both a generic and specific event handler, and then disables/deletes the specific one, the generic one will not fire because the form still handles the specific event.